pipx/commands.py

```diff
@@ -451,12 +498,25 @@ def uninstall(venv_dir: Path, package: str, local_bin_dir: Path, verbose: bool):
             )
         return
 
+    # TODO 20191024: Uninstall injected packages apps in bin
+    # TODO 20191026: Handle error if package is not installed?
     venv = Venv(venv_dir, verbose=verbose)
 
-    metadata = venv.get_venv_metadata_for_package(package)
-    app_paths = metadata.app_paths
-    for dep_paths in metadata.app_paths_of_dependencies.values():
-        app_paths += dep_paths
+    if venv.pipx_metadata.main_package is not None:
+        all_packages = [venv.pipx_metadata.main_package] + list(
+            venv.pipx_metadata.injected_packages.values()
+        )
+        app_paths: List[Path] = []
+        for viewed_package in all_packages:
```

Comment:
Why `viewed` package and not just `package`?

Reply:
There's already `package` as an argument to this function that I didn't want to overwrite.  It was the best other name I could think of.

Reply:
I'm not opinionated, if you have a better name. :)

Reply:
> There's already package as an argument to this function that I didn't want to overwrite. It was the best other name I could think of.

Oh my bad. This is fine.